auslib/web/common/releases.py

```diff
@@ -107,7 +87,7 @@ def get_release_history(release):
                                    order_by=order_by,
                                    get_object_callback=lambda: _get_release(release),
                                    history_filters_callback=_get_filters,
-                                   process_revisions_callback=_process_revisions)
+                                   process_revisions_callback=process_release_revisions)
```

Comment:
Maybe you can pass `annotateRevisionDifferences` directly here and delete 
 `process_release_revisions`

Reply:
Do you mean by stripping the `data` field first, and then calling `annotateRevisionDifferences`, using something like `lambda revisions: annotateRevisionDifferences(map(strip_data, revisions))`? I think that's sufficiently complicated that I'd prefer to keep it as a `def`.

Reply:
Ah, no!
I mean `process_revisions_callback=annotateRevisionDifferences`

Reply:
Does this mean we no longer `strip_data`? Is that OK? Could we get rid of stripping data from other places?

Reply:
Oh, my mistake, I do not realized that line 80 is part of `process_release_revisions`.